cernopendata/modules/fixtures/data/records/phenix-emcal-finding-pi0s-and-photons.json

```diff
@@ -0,0 +1,259 @@
+        "type": "FIXME",
+        "variable": "sec2"
+      },
+      {
+        "description": "“Core” energy of the second cluster",
+        "type": "FIXME",
+        "variable": "Ecore2"
+      },
+      {
+        "description": "Time-of-flight of the second cluster",
+        "type": "FIXME",
+        "variable": "tof2"
+      },
+      {
+        "description": "Number of towers in the second cluster",
+        "type": "FIXME",
+        "variable": "twrhit2"
+      },
+      {
+        "description": "Probability that the second cluster is a photon (based on χ2)",
+        "type": "FIXME",
+        "variable": "prob2"
+      },
+      {
+        "description": "χ2 from expected photon shape for the second cluster",
+        "type": "FIXME",
+        "variable": "chisq2"
+      },
+      {
+        "description": "Combined variable to describe “photonness” of second cluster",
+        "type": "FIXME",
+        "variable": "stoch2"
+      },
+      {
+        "description": "EMCal sector of the cluster (γ-candidate)",
+        "type": "FIXME",
+        "variable": "sec"
+      },
+      {
+        "description": "“Core” energy of the cluster (γ-candidate)",
+        "type": "FIXME",
+        "variable": "ecore"
+      },
+      {
+        "description": "Energy in the central tower of the cluster (γ-candidate)",
+        "type": "FIXME",
+        "variable": "ecent"
+      },
+      {
+        "description": "Time-of-flight in the central tower of the cluster (γ-candidate)",
+        "type": "FIXME",
+        "variable": "tof"
+      },
+      {
+        "description": "Probability that the cluster is a photon (based on χ2)",
+        "type": "FIXME",
+        "variable": "prob"
+      },
+      {
+        "description": "Dispersion of the cluster (γ-candidate)",
+        "type": "FIXME",
+        "variable": "disp"
+      },
+      {
+        "description": "χ2 from expected photon shape of the cluster (γ-candidate)",
+        "type": "FIXME",
+        "variable": "chisq"
+      },
+      {
+        "description": "Number of towers in the cluster (γ-candidate)",
+        "type": "FIXME",
+        "variable": "twrhit"
+      },
+      {
+        "description": "Combined variable to describe “photonness” of the cluster (γ-candidate)",
+        "type": "FIXME",
+        "variable": "stoch"
+      },
+      {
+        "description": "x-position of impact point on the EMCal surface",
+        "type": "FIXME",
+        "variable": "x"
+      },
+      {
+        "description": "y-position of impact point on the EMCal surface",
+        "type": "FIXME",
+        "variable": "y"
+      },
+      {
+        "description": "z-position of impact point on the EMCal surface",
+        "type": "FIXME",
+        "variable": "z"
+      }
+    ],
+    "date_published": "2021",
+    "distribution": {
+      "formats": [
+        "C",
+        "pdf",
+        "root"
+      ],
+      "number_events": 1,
```

Comment:
Total number of events in the four ROOT files is 1,255,308 - listed per file in the new pdf document

Reply:
Thanks, updated.